python_bindings/correctness/float_precision_test.py

```diff
@@ -0,0 +1,18 @@
+import halide as hl
+import numpy
+
+
+def test():
+    def test_pattern(c):
+        x = hl.Var("x")
+        f = hl.Func("f")
+        f[x] = x * c
+        for i, v in enumerate(numpy.asarray(f.realize(10))):
+            assert v == (i * c), "{}[{}]: {} != {}".format(i, c, v, (i * c))
+
+    test_pattern(0.123456789012345678)
```

Comment:
Thanks for adding a test, but I suspect this will be a fragile test; testing exact matches for numeric operations on doubles is usually fraught with flakiness, unless you know you are always running on exactly the same runtime environment. Comparisons of this sort really need to be done with an epsilon of some sort.